lib/tests/cleanup_before.rb

```diff
@@ -48,6 +50,11 @@ def run!(args:)
         cleanup_shared
 
         installed_taps = Tap.select(&:installed?).map(&:name)
+        if on_github_hosted_runner && tap.to_s == CoreTap.instance.name
+          (installed_taps - REQUIRED_TAPS).each do |tap|
+            test "brew", "untap", tap
+          end
+        end
```

Comment:
Won't this untap homebrew/core?

Reply:
Whoops -- I misread `REQUIRED_TAPS` as there was a `CoreTap.instance.name` below.